apps/x509.c

```diff
@@ -72,7 +72,7 @@ const OPTIONS x509_options[] = {
     {"outform", OPT_OUTFORM, 'f',
      "Output format - default PEM (one of DER or PEM)"},
     {"out", OPT_OUT, '>', "Output file - default stdout"},
-    {"keyform", OPT_KEYFORM, 'F', "Private key format - default PEM"},
+    {"keyform", OPT_KEYFORM, 'f', "Private key format - default PEM"},
```

Comment:
'E' looks more appropriate, i.e. PEM, DER or ENGINE (here and also for "cakeyform")